content/docs/validation.mdx

```diff
@@ -54,3 +54,21 @@ For example in React it could look like this:
 Here you can see the different modes in action
 
 <ValidationExample />
+
+## External Validation Errors
+
+External errors (e.g. coming from a backend) can be supplied to the form via the `additionalErrors` prop. These errors are mixed in with the regular errors coming from the AJV validation.
+The prop needs to be updated when necessary, as the [ValidationMode](#validationmode) has no effect on the external errors and only affects the errors coming from AJV. 
```

Comment:
The prop needs to be updated when necessary but this has nothing to do with the `validationMode`. We should probably just split the sentence.

e.g.

Whenever the external errors change, the prop needs to be updated.
Note that the `validationMode` property has no effect on `additionalErrors`, so even when handing over `NoValidation` or `ValidateAndHide` the additional errors will still be shown.